plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -843,6 +849,7 @@ def __init__(self):
             vm_identity=dict(type='str', choices=['SystemAssigned']),
             winrm=dict(type='list'),
             boot_diagnostics=dict(type='dict'),
+            ephemeral_os_disk=dict(type='bool', default=False),
```

Comment:
don't set a default value for ephemeral_os_disk.

Reply:
Ok! I will remove the default value!